tests/src/core/testqgslayoutlabel.cpp

```diff
@@ -44,6 +47,7 @@ class TestQgsLayoutLabel : public QObject
     void evaluation();
     // test expression evaluation when a feature is set
     void feature_evaluation();
+    void feature_evaluation2();
```

Comment:
Maybe rename to make it clearer that one test is using atlas and the other one is not

Reply:
Fixed